modules/exploits/multi/fileformat/archive_tar_arb_file_write.rb

```diff
@@ -0,0 +1,72 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'rex/tar'
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::FILEFORMAT
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'PEAR Archive_Tar < 1.4.11 Arbitary File Write',
```

Comment:
```suggestion
        'Name' => 'PEAR Archive_Tar < 1.4.11 Arbitrary File Write',
```

Reply:
Will be fixed with next commit, thanks for pointing this out!